packages/formation/sass/modules/_m-button.scss

```diff
@@ -112,10 +112,10 @@ button.short {
 
     @include media($medium-large-screen) {
       display: block;
-      height: scale-rem(1.5rem);
+      height: 15px;
       margin-right: scale-rem(0.25rem);
```

Comment:
Thanks @it-harrison! I'm going to keep an eye on what you need to update in Formation and so that I can get it into this [CSS Library PR](https://github.com/department-of-veterans-affairs/component-library/pull/1124) to save us from any regressions.

Did you happen to need that `margin-right` value changed to `px` too?  If so, I'll make sure to update the other PR with it.

```css
margin-right: scale-rem(0.25rem);
```

Reply:
@jamigibbs thanks, I just updated it